lib/travis/api/app/endpoint/repos.rb

```diff
@@ -16,8 +16,13 @@ class Repos < Endpoint
       # json(:repositories)
       get '/' do
         prefer_follower do
-          params['ids'] = params['ids'].split(',') if params['ids'].respond_to?(:split)
-          respond_with service(:find_repos, params)
+          find_repos_service = service(:find_repos, params).run
+          if find_repos_service
+            params['ids'] = params['ids'].split(',') if params['ids'].respond_to?(:split)
```

Comment:
Is this modifying the params that have already been passed into `service(:find_repos)`?